deps/openssl/config/archs/zos-s390x/no-asm/crypto/include/internal/bn_conf.h

```diff
@@ -0,0 +1,28 @@
+/* WARNING: do not edit! */
+/* Generated by Makefile from crypto/include/internal/bn_conf.h.in */
+/*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef HEADER_BN_CONF_H
+# define HEADER_BN_CONF_H
```

Comment:
Suggest to remove the blank space in '# define'.

Reply:
If the #if section is more than a few lines, the `#endif` is usually followed by something like `  // HEADER_BN_CONF_H`

Reply:
Unfortunately, this is a file generated from `deps/openssl/openssl/include/crypto/bn_conf.h.in.`  The other platforms have this as well, for example: `source/node/deps/openssl/config/archs/linux64-s390x/no-asm/crypto/include/internal/bn_conf.h`